### PR TITLE
(#1228) Remove some more static call to assert

### PR DESCRIPTION
**Pull request by iakunin**
This is for https://github.com/yegor256/cactoos/issues/1228

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1229](https://github.com/yegor256/cactoos/issues/1229) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1229](https://github.com/yegor256/cactoos/issues/1229) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/1674f9c2-dd45-44d8-8e5e-d989d118bb0d); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by iakunin** [comment]
@paulodamaso can you refuse it for @fabriciofx so that someone else is assigned?

**Comment by iakunin** [comment]
@fabriciofx could you review this PR, please?

File: src/test/java/org/cactoos/func/BiFuncOfTest.java [hunk 1]
```diff
@@ -39,55 +39,59 @@
...
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Must convert runnable into bi-function",
             new BiFuncOf<String, Integer, Boolean>(
                 () -> done.set(true),
                 true
             ).apply("hello, world", 1),
-            Matchers.equalTo(true)
-        );
+            new IsEqual<>(true)
```
Comment (fabriciofx) [inline src/test/java/org/cactoos/func/BiFuncOfTest.java]: @iakunin please, use `IsTrue` instead.

File: src/test/java/org/cactoos/func/BiFuncOfTest.java [hunk 2]
```diff
@@ -39,55 +39,59 @@
...
     public void convertsValueIntoBiFunc() throws Exception {
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Must convert value into bi-function",
             new BiFuncOf<String, Integer, Boolean>(
                 true
             ).apply("hello, dude!", 1),
-            Matchers.equalTo(true)
-        );
+            new IsEqual<>(true)
```
Comment (fabriciofx) [inline src/test/java/org/cactoos/func/BiFuncOfTest.java]: @iakunin please, use `IsTrue` instead.

File: src/test/java/org/cactoos/func/BiFuncOfTest.java [hunk 3]
```diff
@@ -39,55 +39,59 @@
...
     @Test
     public void convertsScalarIntoBiFunc() throws Exception {
         final Scalar<Boolean> scalar = () -> true;
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Must convert scalar into bi-function",
             new BiFuncOf<Boolean, Boolean, Boolean>(scalar).apply(false, false),
-            Matchers.equalTo(true)
-        );
+            new IsEqual<>(true)
```
Comment (fabriciofx) [inline src/test/java/org/cactoos/func/BiFuncOfTest.java]: @iakunin please, use `IsTrue` instead.

File: src/test/java/org/cactoos/func/FuncOfTest.java [hunk 4]
```diff
@@ -41,32 +40,34 @@
...
     public void convertsValueIntoFunc() throws Exception {
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Must convert value into function",
             new FuncOf<String, Boolean>(
                 true
             ).apply("hello, dude!"),
-            Matchers.equalTo(true)
-        );
+            new IsEqual<>(true)
```
Comment (fabriciofx) [inline src/test/java/org/cactoos/func/FuncOfTest.java]: @iakunin please, use `IsTrue` instead.

File: src/test/java/org/cactoos/func/BiFuncOfTest.java (hunk 1 above)
Comment (iakunin) [reply src/test/java/org/cactoos/func/BiFuncOfTest.java]: @fabriciofx fixed

File: src/test/java/org/cactoos/func/BiFuncOfTest.java (hunk 2 above)
Comment (iakunin) [reply src/test/java/org/cactoos/func/BiFuncOfTest.java]: @fabriciofx fixed

File: src/test/java/org/cactoos/func/BiFuncOfTest.java (hunk 3 above)
Comment (iakunin) [reply src/test/java/org/cactoos/func/BiFuncOfTest.java]: @fabriciofx fixed

File: src/test/java/org/cactoos/func/FuncOfTest.java (hunk 4 above)
Comment (iakunin) [reply src/test/java/org/cactoos/func/FuncOfTest.java]: @fabriciofx fixed

**Comment by iakunin** [comment]
@fabriciofx let's have a look one more time, please

**Comment by fabriciofx** [comment]
@iakunin seems fine to me. @paulodamaso can you merge it, please?

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
The job [#1229](https://github.com/yegor256/cactoos/issues/1229) is now out of scope

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**